packages/webpack/src/utils/async-mfs.js

```diff
@@ -0,0 +1,24 @@
+import MFS from 'memory-fs'
+export default class AsyncMFS extends MFS {}
+
+const syncRegex = /Sync$/
+
+const propsToPromisify = Object.getOwnPropertyNames(MFS.prototype).filter(n => syncRegex.test(n))
+
+for (const prop of propsToPromisify) {
+  const asyncProp = prop.replace(syncRegex, '')
+  const origAsync = AsyncMFS.prototype[asyncProp]
+
+  AsyncMFS.prototype[asyncProp] = function (...args) {
```

Comment:
If it would be better that use `async` for this function ?

Reply:
Good idea :) Makes code cleaner.

Reply:
Oops no we can't. Because of webpack callback support with promisifed version :( MFS async functions  are internally called by webpack too.